core/src/state.rs

```diff
@@ -93,6 +93,13 @@ impl State {
             holochain_public_key,
         ))
     }
+
+    pub fn get_admin_public_key(&self) -> Result<PublicKey, Error> {
```

Comment:
```suggestion
    pub fn admin_public_key(&self) -> Result<PublicKey, Error> {
```

Reply:
Also, signature is wrong, because this can't really ever fail.

```suggestion
    pub fn admin_public_key(&self) -> PublicKey {
```